tests/schemas/valid/test_calculations_value_source.json

```diff
@@ -0,0 +1,81 @@
+{
```

Comment:
The schema name doesn't match the content. This schema has not value source...?

Reply:
I meant value as general "source" for calculations here, renamed to `test_calculations_with_numeric_value.json` or should it be shorter: `test_calculations_numeric_value.json`?